### ISSUE TITLE
`ts-node: command not found` with postinstall script

### ISSUE THREAD

**Issue by peterblazejewicz** [alexjoverm/typescript-library-starter#143]
Hi,

when using current version with Node 9.*/NPM 5.5.1 on macOS I'm getting this during postistall:

```bash
> --libraryname--@0.0.0 postinstall /Users/piotrblazejewicz/git/my-library
> ts-node tools/init

sh: ts-node: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! --libraryname--@0.0.0 postinstall: `ts-node tools/init`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the --libraryname--@0.0.0 postinstall script.
```

That could be `ts-node` issue and could be recently fixed, when I switch to:

```dif
@@ -96,7 +96,7 @@
     "rollup-plugin-sourcemaps": "^0.4.2",
     "semantic-release": "^8.0.0",
     "ts-jest": "^21.0.0",
-    "ts-node": "^3.0.6",
+    "ts-node": "^3.3.0",
     "tslint": "^5.4.3",
     "tslint-config-prettier": "^1.1.0",
     "tslint-config-standard": "^7.0.0",
```

the poinstall script continues and NPM script can execute `ts-node`.
Should this be udpated with PR?
Thanks!

**Comment by alexjoverm** [comment]
Sure, if it is fixed in that way let's do a PR!

**Comment by peterblazejewicz** [comment]
Hm,
should I include `package-lock.json` in the single commit?

**Comment by alexjoverm** [comment]
Fixed by #149
